css/ARN-framework.css

```diff
@@ -0,0 +1,524 @@
+
+  .p-5-child-md div { padding: 5px; }
+
+  .m-r-0-md { margin-right: 0; }
+
+  .vert-mid-md,
+  .vert-mid-md * { vertical-align: middle; align-self: center; }
+
+  .container { max-width: calc(1600px + 10%); padding-left: 0; padding-right: 0; }
+
+  .col { flex-grow: 1; width: auto; }
+  .col-1 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 12) - 21.99px); }
+  .col-2 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 6) - 20px); }
+  .col-2-4-12 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 3) - 16px); }
+  .col-3 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 4) - 18px); }
+  .col-4 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 3) - 16px); }
+  .col-5 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 2.4) - 14px); }
+  .col-6 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 2) - 12px); }
+  .col-7 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.7) - 17px); }
+  .col-8 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.5) - 8px); }
+  .col-9 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.33) - 8.6px); }
+  .col-10 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.2) - 4.3px); }
+  .col-11 { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.09) - 3.4px); }
+  .col-12 { display: block; width: 100%; max-width: calc((var(--maxWidth) / 1)); }
+
+  .col-1-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 12) - 21.99px); }
+  .col-2-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 6) - 20px); }
+  .col-3-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 4) - 18px); }
+  .col-4-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 3) - 16px); }
+  .col-5-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 2.4) - 14px); }
+  .col-6-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 2) - 12px); }
+  .col-7-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.7) - 17px); }
+  .col-8-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.5) - 8px); }
+  .col-9-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.33) - 8.6px); }
+  .col-10-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.2) - 4.3px); }
+  .col-11-md { display: inline-block; width: 100%; max-width: calc((var(--maxWidth) / 1.09) - 3.4px); }
+  .col-12-md { display: block; width: 100%; max-width: calc((var(--maxWidth) / 1)); }
+
+  .bg-jumbo { height: 50vw; }
+}
+
+/* Large devices (laptops/desktops, 1024px and up) */
+@media only screen and (min-width: 1024px) {
+  /*    .bg-responsive{background-color: white;} */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.